finetune/base_models/__init__.py

```diff
@@ -41,6 +41,10 @@ def get_encoder(cls, config=None, **kwargs):
     def get_featurizer(cls, X, encoder, config, train=False, reuse=None, **kwargs):
         return cls.featurizer(X, encoder, config, train=train, reuse=reuse, **kwargs)
 
+    @classmethod
+    def process_base_model(cls):
```

Comment:
I always feel bad being a stickler about naming but I do it anyhow.  Could we call this something more descriptive -- along the lines of "translate_base_model_format"?